simple_clock/app.js

```diff
@@ -1,3 +1,92 @@
-window.addEventListener("load", function() {
-  console.log("Hello World!");
+var timetext;
+var watchtimetext;
+var start;
+var watchtime;
+var elapsed;
+var timoutid;
+var spbutton;
+var rebutton;
+var opbutton;
+var minute;
+var second;
+
+//f1 function
+function showTime() {
+    var now = new Date();
+    timetext.innerHTML = now.toLocaleDateString("zh-TW") + " " + now.toDateString().match("\\D{3}") + " " + now.toTimeString();
```

Comment:
Great, you know how to use regular expression. Just be careful that you are concatenating a string with an array.